addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -59,6 +59,8 @@ const WebsiteWysiwyg = Wysiwyg.extend({
         // Dropdown menu initialization: handle dropdown openings by hand
         var $dropdownMenuToggles = $editableWindow.$('.o_mega_menu_toggle, #top_menu_container .dropdown-toggle');
         $dropdownMenuToggles.removeAttr('data-bs-toggle').dropdown('dispose');
+        // Since bootstrap 5.1.3, removing bsToggle is not sufficient anymore.
+        $editableWindow.$(".o_mega_menu, #top_menu_container .dropdown-menu").addClass("o_wysiwyg_submenu");
```

Comment:
Instead of repeating kinda the same selector, can't you use `$dropdownMenuToggles` to find the menu? Amonst sibling I guess?